samples/apps/logging/js/app.json

```diff
@@ -26,6 +26,26 @@
         "openapi": {}
       }
     },
+    "/log/private/all": {
+      "delete": {
+        "js_module": "logging.js",
+        "js_function": "clear_private",
+        "forwarding_required": "always",
+        "authn_policies": ["jwt", "user_cert"],
+        "mode": "readwrite",
+        "openapi": {}
+      }
+    },
+    "/log/private/count": {
+      "get": {
+        "js_module": "logging.js",
+        "js_function": "count_private",
+        "forwarding_required": "always",
+        "authn_policies": ["jwt", "user_cert"],
+        "mode": "readonly",
+        "openapi": {}
+      }
+    },
```

Comment:
Not particularly RESTful, but a) its 'just' a sample, b) a reasonable extension to what's already there?